docs/_spec/05-classes-and-objects.md

```diff
@@ -186,7 +186,7 @@ A member definition ´M´ _matches_ a member definition ´M'´, if ´M´ and ´M
 
 1. Neither ´M´ nor ´M'´ is a method definition.
 2. ´M´ and ´M'´ define both monomorphic methods with equivalent argument types.
-3. ´M´ defines a parameterless method and ´M'´ defines a method with an empty parameter list `()` or _vice versa_.
+3. ´M´ is defined in Java and defines a parameterless method and ´M'´ defines a method with an empty parameter list `()`, or ´M´ is defined in Java and defines a method with an empty parameter list `()` and ´M'´ defines a parameterless method.
```

Comment:
There's no parameterless method in Java :).

Reply:
Indeed, thank you for the review!